tests/Doctrine/Tests/DBAL/Functional/NamedParametersTest.php

```diff
@@ -10,18 +10,28 @@
  */
 class NamedParametersTest extends \Doctrine\Tests\DbalFunctionalTestCase
 {
-
     public function ticketProvider()
     {
+        $binaryData  = pack("nvc*", 0x1234, 0x5678, 65, 66);
+        $otherBinary = pack("nvc*", 0x1234, 0x5678, 64, 67, 68);
         return array(
             array(
                 'SELECT * FROM ddc1372_foobar f WHERE f.foo = :foo AND f.bar IN (:bar)',
                 array('foo'=>1,'bar'=> array(1, 2, 3)),
-                array('foo'=>PDO::PARAM_INT,'bar'=> Connection::PARAM_INT_ARRAY,),
+                array('foo'=>PDO::PARAM_INT,'bar'=> Connection::PARAM_INT_ARRAY),
+                array(
+                    array('id'=>1,'foo'=>1,'bar'=>1,'baz'=>$binaryData),
+                    array('id'=>2,'foo'=>1,'bar'=>2,'baz'=>$otherBinary),
+                    array('id'=>3,'foo'=>1,'bar'=>3,'baz'=>$binaryData),
+                )
+            ),
+
+            array(
+                'SELECT * FROM ddc1372_foobar f WHERE f.foo = :foo AND f.bar IN (:bar) AND f.baz = :baz',
+                array('foo' => 1,'bar' => array(1, 2, 3), 'baz' => $otherBinary),
+                array('foo'=>PDO::PARAM_INT,'bar'=>Connection::PARAM_INT_ARRAY, 'baz'=>PDO::PARAM_STR),
```

Comment:
This is an example of where I'm currently having difficulties. This line should read:
```php
array('foo'=>PDO::PARAM_INT,'bar'=>Connection::PARAM_INT_ARRAY, 'baz'=>PDO::PARAM_LOB),
```
However that does not work. I am being forced to use STR instead, even though `$otherBinary` and `$binaryData` are both binary strings.